Sources/ClientRuntime/Serialization/SerializationUtils/JSONReadWrite.swift

```diff
@@ -37,4 +51,23 @@ public enum JSONReadWrite {
             try writer.encode(value)
         }
     }
+
+    public static func documentReadingClosure<T: Decodable>(
```

Comment:
Converts data from a successful HTTP response to an output model where the output model is Decodable.